### DISCUSSION TITLE
[Withdrawn] Making a native app for elementary OS in JavaScript

### DISCUSSION THREAD

**Discussion by aral** [elementary/edw#44] [Withdrawn or Not Accepted]
I am planning on porting my little Git commit editor for GNOME ([Gnomit](https://flathub.org/apps/details/org.small_tech.Gnomit)) to a curated elementary OS app for the new app store. It’s written in [GJS](https://gjs.guide/). Given the popularity of JavaScript, I’d be happy to give a talk on the process, if you feel that’d be helpful.

**Comment by orta** [comment]
Hah, I just realized I posted the same thing - https://github.com/elementary/edw/discussions/48

I'll close mine and upvote yours

**Reply by aral** [reply]
Hey, wanna do it together? Love your session description (which is why I want a session on this topic). What if you do the zero-to-app and I can do a brief look at Commit as a real-world app with gotchas, etc.?

**Reply by orta** [reply]
That could work, I can draft out how long it would take to explain some of this stuff over the weekend - might not be a clean 15m 50:50 split but I bet there's some flexibility given it would probably be pre-recorded

**Reply by aral** [reply]
Take as long as you need and let me know when you do. Of course, if the talk is accepted :)

**Comment by aral** [comment]
Assuming since this wasn’t accepted, it was rejected (not sure if talks are still being accepted or not). Regardless, given the lack of communication, I’m withdrawing the submission as there’s no longer enough time for me to prepare for this properly.

**Reply by danirabbit** [reply]
Sorry for the lack of communication! We're still trying to figure out how to organize here since this is our first time. Unfortunately this talk wasn't accepted for this EDW since there were too many similar proposed talks. Please feel free to submit again to a future EDW! Thanks for your time :)

**Comment by aral** [comment]
No worries + good luck with it :)
